prac_05/state_names.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {
```

Comment:
You follow the guidelines for the subject dictionaries "key_to_value". Good Job!